build.sbt

```diff
@@ -2,10 +2,9 @@ import ReleaseTransformations._
 
 ThisBuild / organization := "com.github.dwickern"
 
-lazy val scala3 = "3.0.2"
-lazy val scala213 = "2.13.7"
-lazy val scala212 = "2.12.15"
-lazy val scala211 = "2.11.12"
+lazy val scala3 = "3.6.3"
```

Comment:
Based on my reading https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html#library-maintainers, I think we need to publish for Scala 3.3 LTS. Is that right? It also says I'll need to bump the major version number.

Reply:
Is updated.

My understanding of the documentation is that you need to bump to 4.1.0.

"Remember that when you bump the minor version of the compiler (e.g., moving from the previous Scala LTS line to the next), you also need to bump the minor version of your library." 

The scala version change is from 3.0 to 3.3 which is a minor so the bump here should also be a minor.

Reply:
You're right, I must have misread major instead of minor